dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java

```diff
@@ -332,6 +333,40 @@ public void update(S entry) {
         }
     }
 
+    /**
```

Comment:
Try using a serializer instead to avoid having a json ignored data structure. 
See EntryStarredSerializer for an example.